toolchain/lowering/lowering_handle.cpp

```diff
@@ -215,6 +215,18 @@ auto LoweringHandleStructType(LoweringFunctionContext& /*context*/,
   // No action to take.
 }
 
+auto LoweringHandleTupleValue(LoweringFunctionContext& /*context*/,
+                              SemanticsNodeId /*node_id*/,
+                              SemanticsNode /*node*/) -> void {
+  // TODO.
```

Comment:
```suggestion
  CARBON_FATAL() << "TODO: Add support: " << node;
```
Let's diagnose this error condition, to ensure that lowering doesn't crash when looking for a local value that's not been set.